elkm1_lib/discovery.py

```diff
@@ -87,19 +92,19 @@ class AIOELKDiscovery:
     BROADCAST_ADDRESS = "<broadcast>"
 
     def __init__(self) -> None:
-        self.found_devices: List[ElkSystem] = []
+        self.found_devices: list[ElkSystem] = []
```

Comment:
Be sure to import `from __future__ import annotations` with the new style typing